WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -383,7 +383,7 @@ public static void registerForCloudMessaging(Context context) {
      * select the first one.
      */
     public static Blog getCurrentBlog() {
-        if (currentBlog == null || !wpDB.isDotComBlogVisible(currentBlog.getRemoteBlogId())) {
+        if (currentBlog == null) {
```

Comment:
Do you think that is the best behavior? I am leaning towards creating a new getter for the SitePickerActivity result that does not check for whether the blog is visible.